### PR TITLE
Contacts UI

### PR DESCRIPTION
**Pull request by nkoblai**

### CODE REVIEW

File: moninag/static/src/ts/app/contacts/contacts.service.ts [hunk 1]
```diff
@@ -0,0 +1,61 @@
...
+            second_name: contact.second_name,
+            email: contact.email,
+        });
+
+        return this.http.put(`${this.contactsUrl}${contact['id']}/`, edited_contact, options)
+                        .map((response: Response) => response.json())
+                        .catch((error:any) => Observable.throw(error.json() || 'Server Error'));
+    }
+
+    deleteContact (id:number): Observable<Contact[]> {
```
Comment (lisovyk) [inline moninag/static/src/ts/app/contacts/contacts.service.ts]: Redundant whitespace in `deleteContact (id:number)`

File: moninag/static/src/ts/app/contacts/contacts-email.component.ts [hunk 2]
```diff
@@ -0,0 +1,42 @@
...
+            'first_name': '',
+            'second_name': '',
+            'email':[null, [Validators.required,
+                            Validators.pattern("[A-Z0-9a-z._%+-]+@[A-Za-z0-9.-]+\\.[A-Za-z]{2,64}")]
+      ]})
+    }
+
+    submitForm(contact: any) {
+        this.contactsService.postContact(contact)
+            .subscribe((contact) => {
```
Comment (lisovyk) [inline moninag/static/src/ts/app/contacts/contacts-email.component.ts]: `.subscribe` should be underneath `.postContact`

File: moninag/static/src/ts/app/contacts/contacts-edit.component.ts [hunk 3]
```diff
@@ -0,0 +1,42 @@
...
+
+    ngOnInit(){
+        this.route.params
+            .switchMap((params: Params) => this.contactsService.getContact(+params['id']))
+            .subscribe(contact => this.contact = contact['response']);
+    }
+
+    save(){
+        this.contactsService.putContact(this.contact)
+        .subscribe(() => this.gotoContacts())
```
Comment (lisovyk) [inline moninag/static/src/ts/app/contacts/contacts-edit.component.ts]: `.subscribe` under `.putContact`

File: moninag/static/src/ts/app/contacts/contact.ts [hunk 4]
```diff
@@ -0,0 +1,14 @@
+export class Contact {
+
+    id: number;
+    email: string;
+    first_name: string;
```
Comment (ghost) [inline moninag/static/src/ts/app/contacts/contact.ts:5]: it is strongly suggested to use camelCase for property names and local variables. Here and elsewhere in this pull request.
(According Microsoft typescript style guide https://github.com/Microsoft/TypeScript/wiki/Coding-guidelines)

**Pull request closed without merging**